nixos/modules/services/networking/iperf.nix

```diff
@@ -0,0 +1,92 @@
+{ config, lib, pkgs, ... }: with lib;
+let
+  cfg = config.services.iperf3;
+
+  api = {
+    enable = mkEnableOption "iperf3 network throughput testing server";
+    port = mkOption {
+      type        = types.int;
+      default     = 5201;
+      description = "Server port to listen on for iperf3 client requsts";
+    };
+    affinity = mkOption {
+      type        = types.nullOr types.string;
+      default     = null;
+      description = "CPU affinity for the process";
```

Comment:
You could add an `example = "4,5";` here and explain the syntax in short in the description.

Reply:
The `n,m` syntax is only valid for the client, so I guess this wouldn't make too much sense.